testdata/baselines/reference/submodule/compiler/declarationEmitDoesNotUseReexportedNamespaceAsLocal.types.diff

```diff
@@ -15,7 +15,7 @@
 
  export * as Q from "./sub";
 ->Q : typeof import("sub")
-+>Q : typeof import("./sub.js")
++>Q : typeof import("./sub")
```

Comment:
I think this is something that @weswigham mentioned, that type baselines in Strada did not use real module specifier generation, but they (intentionally?) do in Corsa. That significantly limits the number of diffs that will get completely deleted. IMO, it would be nice to align them and see a pure diff deletion, whether by changing Corsa or changing Strada now that we have a target `tsgo-port` branch.

Reply:
The `typeToTypeNode` call in `typeWriter.ts` provides no symbol tracker, and thus no module resolution host. That's fine, but in such a case, the `createBasicNodeBuilderModuleSpecifierResolutionHost` call inside `checker` that makes a fallback only partially wraps the `host` `program`, and only when `InternalNodeBuilderFlags.DoNotIncludeSymbolChain` is passed. Make that fallback unconditional, and make the wrapper more complete, and you should match the new Corsa behavior.